requirements.txt

```diff
@@ -13,9 +13,10 @@ argparse==1.2.1
 watchdog==0.8.3
 peakutils==1.0.3
 treeherder-client==3.1.0
+matplotlib>=1.5.3
+pandas>=0.19.2
```

Comment:
I agree with @askeing. I think the appveyor is trying to install 2.0.0 on matplotlib. I think it's okay to install using 1.5.3. Also, please try to lock down the version on pandas so that it won't  break as it goes.